src/editorInterface/index.js

```diff
@@ -42,7 +42,8 @@ const isCurrentScopeMarkdownScope = (editor: TextEditor) =>
 
 const isCurrentScopeVueScope = (editor: TextEditor) => getVueScopes().includes(getCurrentScope(editor));
 
-const getCurrentFilePath = (editor: TextEditor) => (editor.buffer.file ? editor.buffer.file.path : undefined);
+const getCurrentFilePath = (editor: TextEditor) =>
+  editor.buffer.file ? editor.buffer.file.getPath() : undefined;
```

Comment:
To fix the Flow error on this line (also reported in CI), you need to update the type defs in [decls/index.js](https://github.com/prettier/prettier-atom/blob/master/decls/index.js).

The [documentation](https://atom.io/docs/api/v1.23.3/File#instance-getPath) indicates that `getPath()` returns a `String`, so the `TextEditor` type should be updated like this:

```js
declare type TextEditor = {
  getGrammar: () => { scopeName: string },
  getBuffer: () => { getRange: () => Range, setTextViaDiff: (text: string) => Range },
  getCursorScreenPosition: () => Point,
  getLastCursor: () => { getScopeDescriptor: () => Atom$ScopeDescriptor },
  getSelectedText: () => string,
  getSelectedBufferRanges: () => Ranges,
  getTextInBufferRange: (bufferRange: Range) => string,
  setCursorScreenPosition: (point: Point) => Point,
  setTextInBufferRange: (bufferRange: Range, text: string) => Range,
  buffer: {
    file: ?{
      path: ?FilePath,
      getPath: () => string // HERE
    }
  },
  backwardsScanInBufferRange: (regex: RegExp, Range: Range, iterator: Atom$Iterator) => void,
};
```

Reply:
Thank you!